src/state/profile/getProfile.ts

```diff
@@ -1,21 +1,61 @@
-import { getPancakeProfileAddress } from 'utils/addressHelpers'
-import abi from 'config/abi/pancakeProfile.json'
+import { getPancakeProfileAddress, getPancakeRabbitsAddress } from 'utils/addressHelpers'
+import pancakeProfileAbi from 'config/abi/pancakeProfile.json'
+import pancakeRabbitsAbi from 'config/abi/pancakeRabbits.json'
 import { getContract } from 'utils/web3'
-import { Profile } from 'state/types'
+import { IpfsData, Profile } from 'state/types'
+import nfts from 'config/constants/nfts'
+import { Nft } from 'config/constants/types'
+import makeBatchRequest from 'utils/makeBatchRequest'
+import { transformProfileResponse, transformTeamResponse, TeamResponse } from './helpers'
 
-const pancakeProfileAddress = getPancakeProfileAddress()
-const contract = getContract(abi, pancakeProfileAddress)
+const profileContract = getContract(pancakeProfileAbi, getPancakeProfileAddress())
+const rabbitContract = getContract(pancakeRabbitsAbi, getPancakeRabbitsAddress())
+const profileApi = process.env.REACT_APP_API_PROFILE
+
+/**
+ * Remove "ipfs://" prefix
+ * e.g. ipfs://QmRcg86epUhTqSXmchjHRzJsXynmLkJqpZCxten4LdX578/flipper.json
+ */
+const stripIpfsPrefix = (uri: string) => uri.substr(6)
 
 const getProfile = async (address: string): Promise<Profile> => {
   try {
-    const hasRegistered = await contract.methods.hasRegistered(address).call()
+    const hasRegistered = await profileContract.methods.hasRegistered(address).call()
 
     if (!hasRegistered) {
       return null
     }
 
-    const profile = await contract.methods.getUserProfile(address).call()
-    return profile
+    const profileResponse = await profileContract.methods.getUserProfile(address).call()
+    const { userId, numberPoints, teamId, tokenId, nftAddress, isActive } = transformProfileResponse(profileResponse)
+    const [tokenURI, teamProfileResponse] = await makeBatchRequest([
+      rabbitContract.methods.tokenURI(tokenId).call,
```

Comment:
I realized that the profile does not save the bunny id so we have to get the bunny information from ipfs